.github/workflows/static-code-analysis.yml

```diff
@@ -0,0 +1,42 @@
+name: "Static code analysis"
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  schedule:
+    # Everyday at midnight
+    - cron: '0  0 * * *'
```

Comment:
Double space `0xx0`. Then we can push.

Reply:
If I understood you correctly, removing a space would be enough, which I have done.